### PR TITLE
Offers links hope

### PR DESCRIPTION
**Pull request by Hopeopolis**
I added about a dozen links to the first two sections of Offers and Negotiations. Want to make sure this all goes as expected before I add too much more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1612,12 +1616,14 @@ that depends greatly on your skills, the marketplace of candidates, what other o
...
 company’s needs.
 However, this section covers some basics of what to expect with offers, and tips on
-negotiating an offer.
+[negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
 
 ### Offers
 
-🔹 Most companies, especially well-established ones, give roughly equal treatment to
-candidates. But even so, sophisticated negotiators can often get better offers.
+🔹 Most companies, especially well-established ones, give roughly equal treatment
```
Comment (jlevy) [inline README.md]: roughly equal treatment duplicated

**Comment by jlevy** [comment]
Adding @RacheltheEditor to review those links too.

File: README.md [hunk 2]
```diff
@@ -1658,7 +1668,8 @@ manager, your future manager, or a recruiter, or some combination) multiple time
 signing. This helps you negotiate details and offers you a chance to get to know the
 people, the company, and the role, so that you can make the best decision for you.
 
-Things to look for in the offer letter include:
+Things to look for in [the offer letter](https://blog.carta.com/a-better-offer-letter/)
```
Comment (jlevy) [inline README.md]: Hm. This is an interesting blog post likely worth linking to, but it's just Carta's way of doing it, not what's typical. Seems like this isn't a clear context around the link.

File: README.md [hunk 3]
```diff
@@ -1612,12 +1616,14 @@ that depends greatly on your skills, the marketplace of candidates, what other o
...
 However, this section covers some basics of what to expect with offers, and tips on
-negotiating an offer.
+[negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
 
 ### Offers
 
-🔹 Most companies, especially well-established ones, give roughly equal treatment to
-candidates. But even so, sophisticated negotiators can often get better offers.
+🔹 Most companies, especially well-established ones, give roughly equal treatment
+[roughly equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/)
```
Comment (RacheltheEditor) [inline README.md]: So this link contains advice to equalize treatment--That's better than what we actually have written here, that most companies already treat people equally, which simply isn't true. Keep the link and change the text. Eg "Companies can and should work hard to ensure that **all candidates are given equal treatment** in the hiring process. But candidates should also know their worth and understand that they can negotiate to get better offers."

File: README.md [hunk 4]
```diff
@@ -1629,8 +1635,11 @@ candidates. But even so, sophisticated negotiators can often get better offers.
   (Granted this may happen down the road.)
 
 - Many companies will give some flexibility during negotiations, letting you indicate
-  whether you prefer higher salary or higher equity.
-- Candidates with competing offers almost always have more leverage and get better offers.
+  whether you prefer
+  [higher salary or higher equity](https://www.investopedia.com/articles/personal-finance/041515/equity-vs-salary-what-you-need-know.asp).
+- Candidates with
+  [competing offers](https://www.shrm.org/resourcesandtools/hr-topics/employee-relations/pages/using-a-job-offer-as-leverage-is-no-longer-a-big-no-no.aspx)
```
Comment (RacheltheEditor) [inline README.md]: Great link!

File: README.md (hunk 2 above)
Comment (RacheltheEditor) [reply README.md]: Agreed. I think it is helpful to offer a specific example, though! Just change the link text to make it clear this is how one company, Carta, does it. "Here's **one example from Carta** of what's included in an offer letter. Think you can expect to see include: ...."

Comment (jlevy) [reply README.md]: The Carta thing is nice, but just not typical (its intent is also marketing for them).

This is closer to what we need. Let's use it instead unless we can find a better one. https://www.glassdoor.com/blog/how-to-read-offer-letter/

File: README.md [hunk 5]
```diff
@@ -1629,8 +1635,11 @@ candidates. But even so, sophisticated negotiators can often get better offers.
   (Granted this may happen down the road.)
 
 - Many companies will give some flexibility during negotiations, letting you indicate
-  whether you prefer higher salary or higher equity.
-- Candidates with competing offers almost always have more leverage and get better offers.
+  whether you prefer
+  [higher salary or higher equity](https://www.investopedia.com/articles/personal-finance/041515/equity-vs-salary-what-you-need-know.asp).
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Yeah, they're different. Let's include both.

File: README.md (hunk 3 above)
Comment (RacheltheEditor) [reply README.md]: (Reminder also that we can forward link here to a later discussion (TK) and links re: negotiating for equity (in treatment and compensation!) as a woman, POC, or other underrepresented employee.)

File: README.md (hunk 2 above)
Comment (Hopeopolis) [reply README.md]: I removed the Carta link and replaced with Glassdoor. Will see if there better links out there and consider further but wanted to get fix in.

**Comment by Hopeopolis** [comment]
Made quick changes per review comments so they are not outstanding but will consider equal treatment/compensation topic and offer letter links more tomorrow when I'll finish adding links to Offers and Negotiations.

**Pull request merged**